spring-cloud-gcp-data-spanner/src/main/java/org/springframework/cloud/gcp/data/spanner/core/convert/ConverterAwareMappingSpannerEntityWriter.java

```diff
@@ -59,16 +60,17 @@
 
 	public static final Map<Class<?>, BiFunction<ValueBinder, ?, ?>> singleItemType2ToMethodMap;
 
-	static final Map<Class<?>, BiConsumer<ValueBinder<?>, Iterable>> iterablePropertyType2ToMethodMap;
+	static final Map<Class<?>, BiConsumer<ValueBinder<Mutation.WriteBuilder>, Iterable>>
+			iterablePropertyType2ToMethodMap = createIterableTypeMapping();
 
-	static {
+	@SuppressWarnings("unchecked")
+	private static Map<Class<?>, BiConsumer<ValueBinder<Mutation.WriteBuilder>, Iterable>> createIterableTypeMapping() {
```

Comment:
@meltsufin I resisted putting these in their own function just to suppress the warning because I suspect this may not be necessary once we go to a later version of Java. In fact, the reason we used a static block instead of just building up the builder using extension methods is because of Java 8 quirks (it worked in Java 9).